README.md

```diff
@@ -61,7 +61,7 @@ using (var reader = new StreamReader(stream))
 Example for rendering to PNG:
 
 ```csharp
-var barcode = QrEncoder.Encode("Hello World!");
+var barcode = QrEncoder.Encode("Hello World!", Barcoder.Qr.ErrorCorrectionLevel.M, Encoding.Auto);
```

Comment:
I think it's better to pass `ErrorCorrectionLevel.M` and `Encoding.Auto` as default values to the [Encode method](https://github.com/huysentruitw/barcoder/blob/master/src/Barcoder/Qr/QrEncoder.cs#L11) instead of changing this line in the readme.